eng/pipelines/extra-platforms/runtime-extra-platforms-maccatalyst.yml

```diff
@@ -77,3 +77,40 @@ jobs:
             creator: dotnet-bot
             interpreter: true
             testRunNamePrefixSuffix: Mono_$(_BuildConfig)
+
+#
+# Build the whole product using Native AOT and run runtime tests
```

Comment:
We should be running sandboxed tests as well, as that is matching more what the end user would experience (sandbox entitlement is required by the App Store).

Reply:
There are failures for the Sandboxed version in the CI, I believe `DevTeamProvisioning` needs to be set to `adhoc` (but not 100% sure).